docs/post-processors/anka-registry-push.mdx

```diff
@@ -38,48 +38,37 @@ Must be `veertu-anka-registry-push`
 #### _**Optional Configuration**_
 
 * `cacert` (String)
-
-Path to a CA Root certificate.
+  * Path to a CA Root certificate.
 
 * `cert` (String)
-
-Path to your node certificate (if certificate authority is enabled).
+  * Path to your node certificate (if certificate authority is enabled).
 
 * `description` (String)
-
-The description of the tag.
+  * The description of the tag.
 
 * `insecure` (Boolean)
-
-Skip TLS verification.
+  * Skip TLS verification.
 
 * `key` (String)
-
-Path to your node certificate key if the client/node certificate doesn't contain one.
+  * Path to your node certificate key if the client/node certificate doesn't contain one.
 
 * `local` (Boolean)
-
-Assign a tag to your local template and avoid pushing to the Registry.
+  * Assign a tag to your local template and avoid pushing to the Registry.
 
 * `registry_path` (String)
-
-The registry URL (will use your default configuration if not set).
+  * The registry URL (will use your default configuration if not set).
 
 * `registry_name` (String)
-
-The registry name (will use your default configuration if not set).
+  * The registry name (will use your default configuration if not set).
 
 * `remote_vm` (String)
-
-The name of a registry template you want to push the local template onto.
+  * The name of a registry template you want to push the local template onto.
 
 * `tag` (String)
-
-The name of the tag to push (will default as 'latest' if not set).
+  * The name of the tag to push (will default as 'latest' if not set).
 
 * `force` (String)
-
-Whether or not to forcefully push, regardless of a tag already existing
+  * Whether or not to forcefully push, regardless of a tag already existing
```

Comment:
Also this is a `(String)` but it looks like it should be a `(Boolean)`. I wasn't sure if I should change it.